# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2013-10-03 09:21+0200\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=CHARSET\n"
"Content-Transfer-Encoding: 8bit\n"

#: usr/lib/ddm/ati.py:72
#, python-format
msgid "ATI card found: %(card)s"
msgstr ""

#: usr/lib/ddm/ati.py:77
#, python-format
msgid "ATI chip serie found: %(serie)s"
msgstr ""

#: usr/lib/ddm/ati.py:86
#, python-format
msgid "ATI driver found: %(drv)s (%(status)s)"
msgstr ""

#: usr/lib/ddm/ati.py:89 usr/lib/ddm/intel.py:56 usr/lib/ddm/nvidia.py:104
#: usr/lib/ddm/via.py:55
#, python-format
msgid "Driver not installable: %(drv)s"
msgstr ""

#: usr/lib/ddm/ati.py:91
#, python-format
msgid "ATI chip serie not supported: %(serie)d"
msgstr ""

#: usr/lib/ddm/ati.py:93
#, python-format
msgid "No driver found for: %(card)s (%(man)s:%(serie)s)"
msgstr ""

#: usr/lib/ddm/ati.py:106 usr/lib/ddm/intel.py:68 usr/lib/ddm/nvidia.py:118
#: usr/lib/ddm/via.py:67
#, python-format
msgid "Install driver: %(drv)s"
msgstr ""

#: usr/lib/ddm/ati.py:120 usr/lib/ddm/nvidia.py:137
#, python-format
msgid "Switch to module: %(module)s"
msgstr ""

#: usr/lib/ddm/ati.py:126 usr/lib/ddm/intel.py:77 usr/lib/ddm/nvidia.py:147
#: usr/lib/ddm/via.py:76
#, python-format
msgid "Driver installed: %(drv)s"
msgstr ""

#: usr/lib/ddm/ati.py:137
msgid "Removing ATI drivers"
msgstr ""

#: usr/lib/ddm/ati.py:151 usr/lib/ddm/intel.py:96 usr/lib/ddm/nvidia.py:176
#: usr/lib/ddm/via.py:95
#, python-format
msgid "Driver removed: %(drv)s"
msgstr ""

#: usr/lib/ddm/ati.py:159
msgid "ATI display driver"
msgstr ""

#: usr/lib/ddm/ati.py:161 usr/lib/ddm/intel.py:104 usr/lib/ddm/nvidia.py:186
msgid "Intel display driver"
msgstr ""

#: usr/lib/ddm/ati.py:163
msgid "Radeon display driver"
msgstr ""

#: usr/lib/ddm/ati.py:165 usr/lib/ddm/intel.py:106 usr/lib/ddm/nvidia.py:190
#: usr/lib/ddm/via.py:105
msgid "Framebuffer display driver"
msgstr ""

#: usr/lib/ddm/ati.py:167 usr/lib/ddm/intel.py:108 usr/lib/ddm/nvidia.py:192
#: usr/lib/ddm/via.py:107
msgid "Vesa display driver"
msgstr ""

#: usr/lib/ddm/ati.py:230
#, python-format
msgid "Install aborted: not in repository: %(repo)s"
msgstr ""

#: usr/lib/ddm/broadcom.py:88
#, python-format
msgid "Broadcom chip serie found: %(chip)s"
msgstr ""

#: usr/lib/ddm/broadcom.py:157
#, python-format
msgid "Used wireless driver: %(drv)s"
msgstr ""

#: usr/lib/ddm/broadcom.py:178
#, python-format
msgid "Broadcom found: %(broadcom)s"
msgstr ""

#: usr/lib/ddm/broadcom.py:201
#, python-format
msgid "Broadcom chipset not supported or ethernet controller: %(broadcom)s"
msgstr ""

#: usr/lib/ddm/broadcom.py:203
#, python-format
msgid "Broadcom chipset not found: %(chip)s"
msgstr ""

#: usr/lib/ddm/broadcom.py:206
#, python-format
msgid "Broadcom pci ID not found: %(id)s"
msgstr ""

#: usr/lib/ddm/broadcom.py:215
#, python-format
msgid "Linux header name to install: %(header)s"
msgstr ""

#: usr/lib/ddm/broadcom.py:219
#, python-format
msgid "Download package: %(package)s"
msgstr ""

#: usr/lib/ddm/broadcom.py:227
#, python-format
msgid "Download package: %(drv)s"
msgstr ""

#: usr/lib/ddm/broadcom.py:264
msgid "Install downloaded packages"
msgstr ""

#: usr/lib/ddm/broadcom.py:272
#, python-format
msgid "Reconfigure package: %(package)s"
msgstr ""

#: usr/lib/ddm/broadcom.py:315
#, python-format
msgid "Purge driver: %(drv)s"
msgstr ""

#: usr/lib/ddm/broadcom.py:325
msgid "Done removing Broadcom drivers"
msgstr ""

#: usr/lib/ddm/ddm.py:81
msgid "Graphics"
msgstr ""

#: usr/lib/ddm/ddm.py:82
msgid "Wireless"
msgstr ""

#: usr/lib/ddm/ddm.py:83
msgid "Kernel"
msgstr ""

#: usr/lib/ddm/ddm.py:84
msgid "Hardware"
msgstr ""

#: usr/lib/ddm/ddm.py:85
msgid "Activated driver"
msgstr ""

#: usr/lib/ddm/ddm.py:86
msgid "Alternative drivers"
msgstr ""

#: usr/lib/ddm/ddm.py:167 usr/lib/ddm/ddm.py:198
msgid "Current drivers are correct"
msgstr ""

#: usr/lib/ddm/ddm.py:208
msgid "PAE already installed"
msgstr ""

#: usr/lib/ddm/ddm.py:210
msgid "PAE support only for 32-bit systems"
msgstr ""

#: usr/lib/ddm/ddm.py:250
msgid "Activate"
msgstr ""

#: usr/lib/ddm/ddm.py:250
msgid "Driver"
msgstr ""

#: usr/lib/ddm/ddm.py:250
msgid "Version"
msgstr ""

#: usr/lib/ddm/ddm.py:250
msgid "Description"
msgstr ""

#: usr/lib/ddm/ddm.py:265
msgid " (recommended)"
msgstr ""

#: usr/lib/ddm/ddm.py:310
msgid "Unknown driver"
msgstr ""

#: usr/lib/ddm/ddm.py:313
msgid ""
"Unknown driver found.\n"
"\n"
"Please remove before installing drivers with DDM."
msgstr ""

#: usr/lib/ddm/ddm.py:358
msgid "Driver activation"
msgstr ""

#: usr/lib/ddm/ddm.py:358
#, python-format
msgid ""
"Are you sure you want to install the following driver:\n"
"\n"
"%(drv)s"
msgstr ""

#: usr/lib/ddm/ddm.py:363 usr/lib/ddm/ddm.py:503
msgid "Locked"
msgstr ""

#: usr/lib/ddm/ddm.py:363
#, python-format
msgid ""
"Could not get lock on %(file)s.\n"
"\n"
"Close any programs locking the file and try again."
msgstr ""

#: usr/lib/ddm/ddm.py:441
msgid "Driver installed"
msgstr ""

#: usr/lib/ddm/ddm.py:441
#, python-format
msgid ""
"Driver installed: %(drv)s\n"
"\n"
"Drivers will be used on next boot.\n"
"Do you want to reboot now?"
msgstr ""

#: usr/lib/ddm/ddm.py:503
#, python-format
msgid ""
"Could not get lock on %(file)s.\n"
"\n"
"Close any programs locking the file and restart DDM."
msgstr ""

#: usr/lib/ddm/ddm.py:554
msgid "Checking your hardware..."
msgstr ""

#: usr/lib/ddm/ddm.py:619
msgid "Start automatic driver install"
msgstr ""

#: usr/lib/ddm/intel.py:47
#, python-format
msgid "Intel card found: %(card)s"
msgstr ""

#: usr/lib/ddm/intel.py:53
#, python-format
msgid "Intel driver found: %(drv)s (%(status)s)"
msgstr ""

#: usr/lib/ddm/main.py:66
#, python-format
msgid "Write debug information to file: %(path)s"
msgstr ""

#: usr/lib/ddm/main.py:70
#, python-format
msgid "Machine info: %(info)s"
msgstr ""

#: usr/lib/ddm/main.py:72
#, python-format
msgid "DDM version: %(version)s"
msgstr ""

#: usr/lib/ddm/main.py:77
msgid "apt-listbugs is installed and might interfere with driver installation"
msgstr ""

#: usr/lib/ddm/main.py:107 usr/lib/ddm/main.py:109
msgid "Please enter your password"
msgstr ""

#: usr/lib/ddm/main.py:115
msgid "DDM - Kernel"
msgstr ""

#: usr/lib/ddm/main.py:116
msgid ""
"You do not have the latest kernel installed.\n"
"\n"
"Use the Update Manager to update your system."
msgstr ""

#: usr/lib/ddm/main.py:120
msgid "DDM - Live environment"
msgstr ""

#: usr/lib/ddm/main.py:121
msgid ""
"DDM cannot run in a live environment\n"
"\n"
"To force start, use the --force argument"
msgstr ""

#: usr/lib/ddm/main.py:125
msgid "DDM - Debian based"
msgstr ""

#: usr/lib/ddm/main.py:126
msgid ""
"Cannot determine the base distribution (debian or ubuntu)\n"
"\n"
"To force start, use the --force argument"
msgstr ""

#: usr/lib/ddm/main.py:130
msgid "DDM - Internet"
msgstr ""

#: usr/lib/ddm/main.py:131
msgid ""
"You do not seem to have an internet connection\n"
"\n"
"To force start, use the --force argument"
msgstr ""

#: usr/lib/ddm/nvidia.py:56
msgid "Update apt"
msgstr ""

#: usr/lib/ddm/nvidia.py:59
msgid "Install nvidia-detect"
msgstr ""

#: usr/lib/ddm/nvidia.py:67
msgid "Cannot install nvidia-detect: abort"
msgstr ""

#: usr/lib/ddm/nvidia.py:91
#, python-format
msgid "Nvidia card found: %(card)s"
msgstr ""

#: usr/lib/ddm/nvidia.py:101
#, python-format
msgid "Nvidia driver found: %(drv)s (%(status)s)"
msgstr ""

#: usr/lib/ddm/nvidia.py:159
msgid "Removing Nvidia drivers"
msgstr ""

#: usr/lib/ddm/nvidia.py:184
msgid "Nvidia display driver"
msgstr ""

#: usr/lib/ddm/nvidia.py:188
msgid "Nouveau display driver"
msgstr ""

#: usr/lib/ddm/pae.py:42
#, python-format
msgid "Installed PAE: %(pae)s"
msgstr ""

#: usr/lib/ddm/pae.py:48
msgid "Multi-core support for 32-bit systems"
msgstr ""

#: usr/lib/ddm/pae.py:58
#, python-format
msgid "Multi-core already installed: %(package)s"
msgstr ""

#: usr/lib/ddm/pae.py:59 usr/lib/ddm/pae.py:72
msgid "PAE capable system"
msgstr ""

#: usr/lib/ddm/pae.py:65
msgid "Multi-core system running single-core kernel found"
msgstr ""

#: usr/lib/ddm/pae.py:69
msgid "PAE not installed"
msgstr ""

#: usr/lib/ddm/pae.py:78
msgid "PAE kernel cannot be installed: single-core system"
msgstr ""

#: usr/lib/ddm/pae.py:94
msgid "Done installing PAE"
msgstr ""

#: usr/lib/ddm/pae.py:108
#, python-format
msgid "PAE package to remove: %(package)s"
msgstr ""

#: usr/lib/ddm/pae.py:111
msgid "Done removing PAE"
msgstr ""

#: usr/lib/ddm/pae.py:113
msgid "Cannot remove PAE when running PAE"
msgstr ""

#: usr/lib/ddm/via.py:46
#, python-format
msgid "Via card found: %(card)s"
msgstr ""

#: usr/lib/ddm/via.py:52
#, python-format
msgid "Via driver found: %(drv)s (%(status)s)"
msgstr ""

#: usr/lib/ddm/via.py:103
msgid "Chrome display driver"
msgstr ""

#: usr/lib/ddm/xorg.py:78
#, python-format
msgid "Used graphics driver: %(module)s"
msgstr ""

#: usr/lib/ddm/xorg.py:100
msgid "No xorg.conf - create minimal xorg.conf"
msgstr ""

#: usr/lib/ddm/xorg.py:120
msgid "New xorg.conf written"
msgstr ""

#: usr/lib/ddm/xorg.py:126
msgid "New section appended to xorg.conf"
msgstr ""
